blog/04-22-2023-interacting-with-smart-contract-on-celo-using-web3py/interacting-with-smart-contract-on-celo-using-web3py.md

```diff
@@ -0,0 +1,1203 @@
+---
+title: Interacting with smart contract on celo using web3py
+description: This tutorial teaches you how to set up and use Web3py - A python-based web3 library
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [celo, solidity, smartcontract]
```

Comment:
```suggestion
tags: ["celosage", "intermediate", "smartcontract", "solidity"]
```

Reply:
Make sure that the tags you use in markdown are the same as the tags you use in users.tsx